python/ray/ml/predictors/integrations/rl/rl_serve_env.py

```diff
@@ -0,0 +1,31 @@
+import gym
```

Comment:
@sven1977 would be great to get your review on this. This is obviously very simplistic, but maybe it's enough for the first iteration? Am I missing any relevant functionality here?